cineast-core/src/main/java/org/vitrivr/cineast/core/util/LogHelper.java

```diff
@@ -7,11 +7,11 @@
 
 public class LogHelper {
 
+  public static final Marker SQL_MARKER = MarkerManager.getMarker("SQL");
+
   private LogHelper() {
   }
 
-  public static final Marker SQL_MARKER = MarkerManager.getMarker("SQL");
```

Comment:
I don't think that marker has been used in years, so it might just as well be removed.